applications/rag/frontend/container/rai/dlp_filter.py

```diff
@@ -14,7 +14,7 @@
 
 import os
 import google.cloud.dlp
-from .retry import retry
+from . import retry
```

Comment:
Why change back to relative imports?

Reply:
retry is under the same level of the folder